app/controllers/concerns/hyrax/dashboard/collections_controller_behavior.rb

```diff
@@ -0,0 +1,279 @@
+module Hyrax
+  module Dashboard
+    module CollectionsControllerBehavior
```

Comment:
I think we made the call to do away with the `*ControllerBehavior` pattern as now Ruby has `Module#prepend` which can be used to achieve the same goals. This also removes some indirection from the gem.